lnbits/middleware.py

```diff
@@ -210,3 +210,18 @@ async def block_allow_ip_middleware(request: Request, call_next):
         return await call_next(request)
 
     app.middleware("http")(block_allow_ip_middleware)
+
+
+def add_first_install_middleware(app: FastAPI):
+    @app.middleware("http")
+    async def first_install_middleware(request: Request, call_next):
+        if request.url.path != "/api/v1/auth/first_install":
+            if (
+                settings.first_install
+                and request.url.path != "/first_install"
+                and not request.url.path.startswith("/static")
+            ):
+                return RedirectResponse("/first_install")
```

Comment:
```suggestion
        if (
            settings.first_install
            and request.url.path != "/api/v1/auth/first_install"
            and request.url.path != "/first_install"
            and not request.url.path.startswith("/static")
        ):
            return RedirectResponse("/first_install")
```

Reply:
- do we have underscore in path in other places?
     - `/api/v1/auth/first_install`
 - suggestion:
      - `/api/v1/auth/init`